prowler/providers/aws/services/ec2/ec2_transitgateway_auto_accept_vpc_attachments/ec2_transitgateway_auto_accept_vpc_attachments.py

```diff
@@ -6,11 +6,7 @@ class ec2_transitgateway_auto_accept_vpc_attachments(Check):
     def execute(self):
         findings = []
         for tgw_arn, tgw in ec2_client.transit_gateways.items():
```

Comment:
I think now `tgw_arn` won't be used anymore. Review if this is being filled in the `check_report`.